test-case/check-sof-logger.sh

```diff
@@ -61,33 +61,34 @@ sleep 2
 dlogc "sudo pkill -9 $(basename $loggerBin)"
 sudo pkill -9 $(basename $loggerBin) 2> /dev/null
 
+func_logger_exit()
+{
+    local file=$1 code=$2 type=${3:-data}
+    dlogi "Log $type BEG>>"
+    cat $file
+    dlogi "<<END $type data"
+    mv /tmp/$rdnum.*.log $LOG_ROOT
```

Comment:
why need to use random name for this logger files? We have a workspace for each test case, no need random name now. Just save it in the LOG_DIR is OK